src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
> what's the problem by adding/removing at runtime besides the null check (and race conditions when subscribing/unsubscribing)

neither of those. busconf is intended for config time only. so i dont think we should should expose event on that.

basically my concern is mixing config and run time.